### PR TITLE
Feat/contact page

### PR DESCRIPTION
**Pull request by montanizstills**

### CODE REVIEW

File: src/app/modules/webapp-container/pages/contact-page/contact-page.component.html [hunk 1]
```diff
@@ -1 +1,27 @@
...
+                </div>
+                <div>
+                    <p class="errorMessage" *ngIf="supportForm.get('email').hasError('pattern')">Please enter a valid email.</p>
+                    <input formControlName="email" class="supportTicketInfo inputField" type="text" placeholder="Your Email" pattern="[A-Za-z0-9]+@[A-Za-z0-9]+(\.[a-z]{2,}){}"  required pInputText/>
+                </div>
+                <div>
+                    <p class="errorMessage" *ngIf="supportForm.get('message').hasError('minlength')">Please provid more information about your issue.</p>
+                    <textarea formControlName="message" class="supportTicketInfo inputField" type="text" placeholder="Message" minlength="80" [rows]="5" [cols]="100" maxlength="700" autoResize="autoResize" required pInputTextarea></textarea>
+                </div>
+                <button id="submitBtn" type="submit" icon="pi pi-envelope" [disabled]="supportForm.invalid" pButton ></button>
```
Comment (ghost1497) [inline src/app/modules/webapp-container/pages/contact-page/contact-page.component.html:20]: Refactor to a input tag to inhert styles like ` <input id="submitBtn" type="submit" [disabled]="supportForm.invalid" >` or add button style to web app container component styles (attachment on story to assist with this) but remove the `pButton` attribute

File: src/app/modules/webapp-container/pages/contact-page/contact-page.component.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+#formContainer{
+    // display: block;
+    border-color: black;
+}
+
+#submitBtn{
+    width: 10em;
+    height: 3em;
+}
```
Comment (ghost1497) [inline src/app/modules/webapp-container/pages/contact-page/contact-page.component.scss:23]: add `@import "../../webapp-container.component.scss";`

File: src/app/modules/webapp-container/pages/contact-page/contact-page.component.scss [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    color: red;
+}
+.inputField{
+    border-color: black;
+    margin: 0.03em;
+    border-radius: 0.3em;
+}
+
+#formContainer{
+    // display: block;
```
Comment (ghost1497) [inline src/app/modules/webapp-container/pages/contact-page/contact-page.component.scss:16]: align the form elements and text to the center

File: src/app/modules/webapp-container/pages/contact-page/contact-page.component.ts [hunk 4]
```diff
@@ -8,13 +12,28 @@ import {BreadcrumbService} from "../../../../services/breadcrumb.service";
...
+    this.supportForm = this.formBuilder.group({
+      firstName:[''],
+      lastName: [''],
+      email: [''],
+      message: ['']
+    });
+  }
+  
+
+  onSubmit(supportForm: FormGroup){
```
[the remark this answers is not in the document]
Comment (montanizstills) [reply src/app/modules/webapp-container/pages/contact-page/contact-page.component.ts:32]: a subscriber?

**Pull request merged**